### PR TITLE
💻 Runtime Benchmarks

### PR DESCRIPTION
**Pull request by JuaniRios**
## What?
Benchmark all the pallets of the runtime again

## Why?
New SDK and simplifications mean different weights

## How?
Run them on the AWS server

## Testing?
Subweight is not working for some reason, so we need to review manually

### CODE REVIEW

**Comment by JuaniRios** [comment]
* **#384** <a href="https://app.graphite.dev/github/pr/Polimec/polimec-node/384?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @JuaniRios and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: runtimes/polimec/src/weights/pallet_preimage.rs [hunk 1]
```diff
@@ -36,19 +35,19 @@ impl<T: frame_system::Config> pallet_preimage::WeightInfo for WeightInfo<T> {
...
+	/// Proof: `Balances::Holds` (`max_values`: None, `max_size`: Some(175), added: 2650, mode: `MaxEncodedLen`)
 	/// Storage: `Preimage::PreimageFor` (r:0 w:1)
 	/// Proof: `Preimage::PreimageFor` (`max_values`: None, `max_size`: Some(4194344), added: 4196819, mode: `MaxEncodedLen`)
 	/// The range of component `s` is `[0, 4194304]`.
 	fn note_preimage(s: u32, ) -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `135`
-		//  Estimated: `4414`
-		// Minimum execution time: 38_290_000 picoseconds.
-		Weight::from_parts(161_037_588, 0)
```
Comment (JuaniRios) [inline runtimes/polimec/src/weights/pallet_preimage.rs:48]: Is this 5x increase normal?

File: runtimes/polimec/src/weights/pallet_utility.rs [hunk 2]
```diff
@@ -36,48 +35,48 @@ impl<T: frame_system::Config> pallet_utility::WeightInfo for WeightInfo<T> {
 		// Proof Size summary in bytes:
 		//  Measured:  `0`
 		//  Estimated: `0`
-		// Minimum execution time: 3_500_000 picoseconds.
-		Weight::from_parts(712_845, 0)
```
Comment (JuaniRios) [inline runtimes/polimec/src/weights/pallet_utility.rs:40]: check this

Comment (lrazovic) [reply runtimes/polimec/src/weights/pallet_utility.rs:40]: Seems in line with other runtimes.

File: runtimes/polimec/src/weights/pallet_preimage.rs (hunk 1 above)
Comment (lrazovic) [reply runtimes/polimec/src/weights/pallet_preimage.rs:48]: On the Collectives System Chain: https://github.com/polkadot-fellows/runtimes/blob/cf15699e1527f3bf4c76abd0287e62721bee428c/system-parachains/collectives/collectives-polkadot/src/weights/pallet_preimage.rs#L59-L70

It's definitely a strange increase, but the final weight is still ok.

File: runtimes/polimec/src/weights/pallet_preimage.rs [hunk 3]
```diff
@@ -82,11 +81,11 @@ impl<T: frame_system::Config> pallet_preimage::WeightInfo for WeightInfo<T> {
 		// Proof Size summary in bytes:
 		//  Measured:  `106`
 		//  Estimated: `3556`
-		// Minimum execution time: 11_990_000 picoseconds.
-		Weight::from_parts(1_776_554, 0)
```
[the remark this answers is not in the document]
Comment (lrazovic) [reply runtimes/polimec/src/weights/pallet_preimage.rs:86]: This seems in line with the Collective System chain: https://github.com/polkadot-fellows/runtimes/blob/cf15699e1527f3bf4c76abd0287e62721bee428c/system-parachains/collectives/collectives-polkadot/src/weights/pallet_preimage.rs#L97

**Review by lrazovic — APPROVED** [review]

**Pull request merged**